### PR TITLE
docs: v1.20 - autoscaling cooldown support

### PR DESCRIPTION
**Pull request by CaptainCarpensir**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v120.en.md [hunk 1]
```diff
@@ -163,6 +163,43 @@ command to create a pipeline [`manifest.yml`](../docs/manifest/pipeline.en.md) f
     ✔ Successfully deployed pipeline: env-pipeline
     ```
 
+## Autoscaling Cooldown Support
```
Comment (dannyrandall) [inline site/content/blogs/release-v120.en.md:173]: add it to the top section as well! with a link down to here

File: site/content/blogs/release-v120.en.md [hunk 2]
```diff
@@ -163,6 +163,43 @@ command to create a pipeline [`manifest.yml`](../docs/manifest/pipeline.en.md) f
...
+A small addition to our service manifests: the ability to configure autoscaling cooldown periods.
+For `Load Balanced`, `Backend`, and `Worker` Services, you can now configure this for their corresponding autoscaling fields.
+
+To configure the cooldown field, your service needs some extra fields to allow the scaling to begin.
+First it needs a `range` of the number of ECS clusters your service can run, and then it needs at least one autoscaling field such as `cpu_percentage` to set a target which the service will scale to meet.
```
Comment (dannyrandall) [inline site/content/blogs/release-v120.en.md]:
```suggestion
First it needs a `range` of the number of ECS clusters your service can run, and then it needs at least one autoscaling field such as `cpu_percentage` to set a target the service will scale to meet.
```

File: site/content/blogs/release-v120.en.md [hunk 3]
```diff
@@ -163,6 +163,43 @@ command to create a pipeline [`manifest.yml`](../docs/manifest/pipeline.en.md) f
...
+    ```
+    count:
+      range: 1-10
+      cooldown:
+        in: 30s
+        out: 30s
+      cpu_percentage: 50
+    ```
+
+This also gives you the option to specify the cooldowns for an individual field.
```
Comment (dannyrandall) [inline site/content/blogs/release-v120.en.md]:
```suggestion
You also have the option to specify the cooldowns for an individual field.
```

File: site/content/blogs/release-v120.en.md (hunk 2 above)
Comment (efekarakus) [inline site/content/blogs/release-v120.en.md]: I wonder if we can prune any of the text to make it a bit more terse, what do you think of something like?
> You can now configure the autoscaling fields under [`range`](../docs/manifest/lb-web-service.en.md#count-rage) to have custom cooldown periods.
> Previously, each scaling metric such as `cpu_percentage` has a set in cooldown of X secs and out cooldown of Y seconds. Now you can set a global cooldown period:
>  \# your yaml
> Alternatively, you can set individual cooldowns:

**Review by efekarakus — APPROVED** [review]
Thank you Aiden ! 😍

File: site/content/blogs/release-v120.en.md [hunk 4]
```diff
@@ -163,6 +163,39 @@ command to create a pipeline [`manifest.yml`](../docs/manifest/pipeline.en.md) f
     ✔ Successfully deployed pipeline: env-pipeline
     ```
 
+## Autoscaling Cooldown Support
+A small addition to our service manifests: the ability to configure autoscaling cooldown periods.
+For `Load Balanced`, `Backend`, and `Worker` Services, you can now configure their autoscaling fields under `count` to have custom cooldown periods.
+Previously, each scaling metric such as `cpu_percentage` has a set in cooldown of 120 secs and out cooldown of 60 seconds. Now you can set a global cooldown period:
```
Comment (huanjani) [inline site/content/blogs/release-v120.en.md]: not sure if this was meant?
```suggestion
Previously, each scaling metric such as `cpu_percentage` had a fixed 'in' cooldown of 120 secs and 'out' cooldown of 60 seconds. Now, you can set a global cooldown period:
```

**Review by dannyrandall — APPROVED** [review]

**Pull request merged**